source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -868,56 +874,13 @@ export const handleHardwareWalletRequests = async (
       throw e;
     }
   });
-  // @ts-ignore ts-migrate(2345) FIXME: Argument of type '(params: TrezorSignTransactionRe... Remove this comment to see the full error message
-  signTransactionTrezorChannel.onRequest(async (params) => {
-    const {
-      inputs,
-      outputs,
-      protocolMagic,
-      fee,
-      ttl,
-      networkId,
-      certificates,
-      devicePath,
-      validityIntervalStartStr,
-      withdrawals,
-      auxiliaryData,
-      signingMode,
-    } = params;
-
-    if (!TrezorConnect) {
```

Comment:
Reduntant code - this evaluates to `false` since `TrezorConnect` is an object, and object is always truthy.